src/components/LanguageEngagementListItemCard/LanguageEngagementListItem.graphql

```diff
@@ -42,6 +42,13 @@ fragment LanguageEngagementListItem on LanguageEngagement {
     presetInventory {
       value
     }
+    marketingCountry {
+      canRead
+      canEdit
+      value {
+        ...DisplayLocation
+      }
+    }
```

Comment:
same here. Adding this fixed a bug so that when a languages sensitivity is changed the marketingCountry DataButton will update to show either the Region or the Country based on its sensitivity.